Resources/Locale/en-US/escape-menu/ui/options-menu.ftl

```diff
@@ -37,9 +37,13 @@ ui-options-hud-theme = HUD Theme:
 ui-options-hud-theme-default = Default
 ui-options-hud-theme-modernized = Modernized
 ui-options-hud-theme-classic = Classic
-ui-options-vp-stretch = Stretch viewport to fit game window?
+ui-options-vp-stretch = Stretch viewport to fit game window
 ui-options-vp-scale = Fixed viewport scale: x{ $scale }
-ui-options-vp-low-res = Low-resolution viewport?
+ui-options-vp-integer-scaling = Integer scaling (might cause black bars/clipping)
+ui-options-vp-integer-scaling-tooltip = If this option is enabled, the viewport will be scaled using an integer value.
+                                        While this results in crisp textures, it also often means that black bars appear
+                                        at the top/bottom of the screen or that part of the viewport is not visible.
```

Comment:
This should be **prefer** integer scaling. This description makes it it out as if it forces it on which is absolutely not the case. And update the tooltip.